docs/application/dotnet/guides/nui/relative-layout.md

```diff
@@ -0,0 +1,47 @@
+| `TopRelativeOffset`    | float           | Gets or sets the relative offset for top target. <br /> When value is 0 the top edges of the top target and child view are aligned.<br/> When value is 1 the top edge of the child view is aligned to the bottom edge of the top target.
+| `BottomRelativeOffset` | float           | Gets or sets the relative offset for bottom target. <br /> When value is 0 the bottom edge of the child view is aligned to the top edge of the right target.<br/>When value is 1 the bottom edges of the bottom target and child view are aligned.
+| `BottomRelativeOffset` | float           | Gets or sets the relative offset for bottom target. <br /> When value is 0 the bottom edge of the child view is aligned to the top edge of the right target.<br/>When value is 1 the bottom edges of the bottom target and child view are aligned.
+| `HorizontalAlignment`  | RelativeLayout.Alignment| Gets or sets the horizontal alignment of this child view.|
+| `VerticalAlignment`  | RelativeLayout.Alignment| Gets or sets the vertical alignment of this child view.|
+| `FillHorizontal`  | bool | Gets or sets the boolean value whether child fills its horizontal space.|
+| `Fillvertical`  | bool | Gets or sets the boolean value whether child fills its vertical space.|
+
+<a name="targetandrelativeoffset"></a>
+## Target And RelativeOffset
+ `Target` and `RelativeOffset` properties determine a size for layout space.
+
+ `Target` indicates the target view that each edge of layout space is aligned with. the default is `null` that means parent relative layout.
+
+ `RelativeOffset` moves a each edge of layout space in proportion to target view size.<br/>
+When the value is `0.0f`, edge of layout space is aligned with left/top edge of target view.<br/>
+When the value is `1.0f`,  edge of layout space is aligned with right/bottom edge of target view.
+
+![RelativeLayoutTargetRelativeOffset](./media/relativeLayoutTargetRelativeOffset.PNG)
+
+<a name="alignment"></a>
+## Alignment
+ `HorizontalAlignment` and `VerticalAlignment` properties describe how to align the child view with layout space. the default is `Start`, for both axes.
+ |Start|Center|End|
+ |:---:|:---:|:---:|
+ |![RelativeAlignStart](./media/relativeLayoutAlignStart.PNG)|![RelativeAlignCenter](./media/relativeLayoutAlignCenter.PNG)|![RelativeAlignEnd](./media/relativeLayoutAlignEnd.PNG)|
+ 
+ <a name="fill"></a>
+## Fill
+ `FillHorizontal` and `FillHorizontal` properties are a boolean value that specifies whether to fill the layout space. the default is `false`, for both axes.
+ |false|true|
+ |:---:|:---:|
+ |![RelativeAlignCenter](./media/relativeLayoutAlignCenter.PNG)|![RelativeFill](./media/relativeLayoutFill.PNG)|
```

Comment:
I would suggest using a small letter extension .png as sometimes, the site does not render the image with capital .PNG.